pkg/util/configmap.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2022 The OpenFunction Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-logr/logr"
+	"github.com/openfunction/pkg/constants"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func GetDefaultConfig(ctx context.Context, c client.Client, log logr.Logger) map[string]string {
```

Comment:
@shyupc Thanks for your contribution. I think we can move GetDefaultConfig to `util/util.go` instead of putting it into a new file `util/configmap.go`